pkg/ebpf/event_parameters.go

```diff
@@ -0,0 +1,97 @@
+package ebpf
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/aquasecurity/tracee/pkg/ebpf/probes"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/filters"
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+type eventParameterHandler func(t *Tracee, eventParams []map[string]filters.Filter[*filters.StringFilter]) error
+
+var eventParameterHandlers = map[events.ID]eventParameterHandler{
+	events.SuspiciousSyscallSource: attachSuspiciousSyscallSourceProbes,
```

Comment:
Since you created event parameters, we can move print_mem_dump in here as well (I'm not sure if there are other events that abuse data filters as parameters)

Reply:
`print_mem_dump` uses event filters to populate the required ksymbols. `handleEventParameters` is called at the end of initialization, because it needs to happen after `initBPF`, which is called after the kernel symbols are initialized.